src/server/android/android.ts

```diff
@@ -297,6 +295,25 @@ export class AndroidDevice extends EventEmitter {
   }
 }
 
+export class AndroidAdbSocket extends EventEmitter {
```

Comment:
I don't think you need this class, just use backend socket.

Reply:
Do you mean to move the logic into the BufferedSocketWrapper class since it implements the SocketBackend interface?

Reply:
What logic?